bin/cron/report_activejob_metrics

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env ruby
+require_relative 'only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+require_relative '../../dashboard/config/environment'
+
+METRICS_NAMESPACE = 'CDO/ActiveJob'
+
+def push_queue_length_metric
+  job_count = Delayed::Job.count
+
+  Cdo::Metrics.push(METRICS_NAMESPACE,
+    [
+      {
+        metric_name: 'JobCount',
+        value: job_count,
+        unit: 'Count',
+        timestamp: Time.now,
+        dimensions: [
+          {
+            name: 'Environment',
+            value: CDO.rack_env # is this the right value?
```

Comment:
I’d prefer a value that uniquely identifies a given deployment. Specifically the value set in the CloudFormation Parameter `SubdomainName` https://github.com/code-dot-org/aiproxy/blob/76eb0b0b9f1ddef78963b078343ba9a00fb622e7/cicd/3-app/aiproxy/template.yml#L14-L16

Reply:
oops … we’re over in the main `code-dot-org` repository, so `DashboardSubdomainName` is what I’d want to use https://github.com/code-dot-org/code-dot-org/blob/77d377d367f650344c56f3257049415a294b5407/aws/cloudformation/cloud_formation_stack.yml.erb#L17-L20

We don’t have an easy way to pass that value into ` CDO.` configuration setting yet. I’ve always wanted to implement `!StackParameter` directive to do that, just like our `!Secret` and `!StackSecret` systems.

The hacky way to do that now is to provision a Stack-specific Secret in the template whose value is populated from a Parameter.

Reply:
Example of using a Secret to populate a non-secret configuration setting https://github.com/code-dot-org/code-dot-org/blob/77d377d367f650344c56f3257049415a294b5407/aws/cloudformation/components/database.yml.erb#L88-L93

Reply:
`CDO.rack_env` does not uniquely identify a specific environment. There are currently 3 different types of environments that have `CDO.rack_env` set to the value `test` (managed test server, an engineer’s workstation when executing unit tests, and continuous integration builds). Would all 3 types of environments publish the same Metric?

Reply:
We're [already](https://github.com/code-dot-org/code-dot-org/blob/2fa69d96a0f7e04970bfce9b0e47f0a94b08271e/dashboard/app/jobs/evaluate_rubric_job.rb#L87) reporting the environment using `rack_env` in relation to this job processing.

I think `rack_env` is what I want, with the exception of the case where we're running tests locally.